Cython/Includes/cpython/ref.pxd

```diff
@@ -48,3 +48,13 @@ cdef extern from "Python.h":
     # It is a good idea to use this macro whenever decrementing the
     # value of a variable that might be traversed during garbage
     # collection.
+
+    Py_ssize_t Py_REFCNT(PyObject* o)
```

Comment:
I'm a bit undecided if this should take a `PyObject*` or an `object` argument.

I don't think it can ever be `NULL` so `object` might be most consistent. It also fits better with most of Cython's internal usages (but less well with the usages in the docs).

I'm not sure there's a perfect answer either way, so I'm happy to be argued with, but I'm leaning towards `object` being better.